deployment/config.go

```diff
@@ -102,7 +102,7 @@ type TerraformDBSettings struct {
 	// Number of DB instances.
 	InstanceCount int `default:"1" validate:"range:[0,)"`
 	// Type of the DB instance.
-	InstanceType string `default:"db.r4.large" validate:"notempty"`
+	InstanceType string `default:"db.r6g.large" validate:"notempty"`
```

Comment:
Was this meant to go in? As far as I understand we are waiting for the reference architecture to decide which instance class to default to going forward.

Reply:
No we already changed the defaults in deployer.sample.json. This got missed out. So yes, this is intended.